azure-pipelines-product-construction-service.yml

```diff
@@ -12,6 +12,7 @@ pr:
       - main
 
 variables:
+- group: Publish-Build-Assets
```

Comment:
The var group is imported always while the variable we use from it is limited to production only. Shouldn't we have it under the same condition?